doc/nrf/releases/release-notes-1.9.0.rst

```diff
@@ -0,0 +1,890 @@
+
+    * Removed:
+
+      * Overview diagram in :ref:`nrfxlib:nrf_modem`.
+      * AT socket page.
+      * Extensions page.
+
+  * :ref:`nrf_security`:
+
+    * Added experimental support for PSA crypto drivers in :ref:`nrf_security`.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+Unity
+-----
+
+* Fixed a bug that resulted in the mocks for some functions not having the ``__wrap_`` prefix. This happened for functions declared with whitespaces between identifier and parameter list.
+
+HID Configurator for nRF Desktop
+--------------------------------
+
+* Updated:
+
+  * The script to recognize the bootloader variant as a DFU module variant for the configuration channel communication. The new implementation is backward compatible - the new version of the script checks for module name and acts accordingly. See the Performing DFU section in :ref:`nrf_desktop_config_channel_script` documentation.
+
+  * Documentation on setting up the dependencies for Debian/Ubuntu/Linux Mint operating system.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``1eedec3e79``, plus some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+* Added:
+
+  * Check of reset address in the incoming image validation phase. See :kconfig:`CONFIG_MCUBOOT_VERIFY_IMG_ADDRESS`.
+
+* Updated:
+
+  * Always call :c:func:`sys_clock_disable` in main since the empty :c:func:`sys_clock_disable` callback is provided if the platform does not support system clock disable capability.
+  * Bootutil library to close flash_area after failure to read swap status information.
+  * Allow image header bigger than 1 KB for encrypted images.
+  * Zephyr:RSA to use a smaller SHA256 implementation.
+
+* Fixed:
+
+  * Zephyr/boot_serial_extension build failure on :c:macro:`MCUBOOT_LOG_MODULE` when LOG is disabled.
+  * Status offset calculation in case of scratch area.
+  * Build with image encryption using RSA (issue introduced by migration to MbedTLS 3.0.0).
+  * Support for single application with serial recovery.
+
+Mcumgr
+======
+
+The mcumgr library contains all commits from the upstream mcumgr repository up to and including snapshot ``657deb65``.
+
+The following list summarizes the most important changes inherited from upstream mcumgr:
+
+* |no_changes_yet_note|
+
+Zephyr
+======
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each NCS release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``3f82656``, plus some |NCS| specific additions.
```

Comment:
@SebastianBoe and please provide a commit here.

Reply:
45ef0d2aef9bbba92460e19129b35f0da9729610 is the correct one

Reply:
I can confirm it is 45ef